components/core/View.tsx

```diff
@@ -38,6 +38,9 @@ interface ViewStyleProps {
   flexGrow?: RNViewStyle['flexGrow'];
   flexShrink?: RNViewStyle['flexShrink'];
   flexWrap?: RNViewStyle['flexWrap'];
+  gap?: RNViewStyle['gap'];
```

Comment:
This seems like a leaky abstraction - do you have any suggestions for how we might improve it in the future?

Reply:
`StyleSheet` is probably worth leveraging. But I think we could really benefit from using `styled-components` or `emotion` which both have `react-native` integrations.